zcash_client_sqlite/src/wallet.rs

```diff
@@ -2037,11 +2033,15 @@ pub(crate) fn get_unspent_transparent_output(
     let mut stmt_select_utxo = conn.prepare_cached(
         "SELECT u.prevout_txid, u.prevout_idx, u.script, u.value_zat, u.height
          FROM utxos u
-         LEFT OUTER JOIN transactions tx
-         ON tx.id_tx = u.spent_in_tx
          WHERE u.prevout_txid = :txid
          AND u.prevout_idx = :output_index
-         AND tx.block IS NULL",
+         AND u.id NOT IN (
+            SELECT txo_spends.id
```

Comment:
Bug: this should be:
```suggestion
            SELECT txo_spends.transparent_received_output_id
```

Bug found by Lukas while testing the Zashi shield funds button.